modules/nf-core/motus/downloaddb/main.nf

```diff
@@ -36,4 +36,16 @@ process MOTUS_DOWNLOADDB {
         motus: \$(grep motus db_mOTU/db_mOTU_versions | sed 's/motus\\t//g')
     END_VERSIONS
     """
+
+    stub:
+    def args     = task.ext.args ?: ''
+    def software = "${motus_downloaddb_script.simpleName}_copy.py"
+
+    """
+    cat <<-END_VERSIONS > versions.yml
```

Comment:
You've not actually made any file in the stub.

Reply:
I updated this.